### DISCUSSION TITLE
nhentai搜什么都是没有找到对应本子

### DISCUSSION THREAD

**Discussion by YipSzeho** [Tsuk1ko/CQ-picfinder-robot#394] [问题求助] [answered]
代理已经设置过了，在nh找的封面让bot搜还是搜不到

**Comment by Tsuk1ko** [comment, marked as answer]
nh 套了 cf，有五秒盾，目前搜索不了，把 `bot.getDoujinDetailFromNhentai` 设为 `false` 可以避免去搜索

**Reply by Tsuk1ko** [reply]
不过我之前好像一直没想到可以去镜像站搜，周末看看好了……

**Reply by YipSzeho** [reply]
好・∀・

**Reply by Tsuk1ko** [reply]
最新版加了个配置 `bot.getDoujinDetailFromNhentaiMirrorSite`，默认去 nhentai.xxx 搜索
